src/app/components/RegisterValidatorHome/components/SelectOperators/components/OperatorSelector/OperatorSelector.tsx

```diff
@@ -1,25 +1,27 @@
 import React, { useEffect, useState } from 'react';
 import { observer } from 'mobx-react';
+import { sha256 } from 'js-sha256';
+import { Grid } from '@material-ui/core';
 import Select from '@material-ui/core/Select';
 import MenuItem from '@material-ui/core/MenuItem';
 import InputLabel from '@material-ui/core/InputLabel';
 import FormControl from '@material-ui/core/FormControl';
+import VerifiedUserIcon from '@material-ui/icons/VerifiedUser';
 import { useStores } from '~app/hooks/useStores';
 import { useStyles } from './OperatorSelector.styles';
```

Comment:
Relative imports should be the last in all the imports.
First goes global imports, then imports from root of the project, then last goes relative ones.